webcomponents/tool/src/main/frontend/.eslintrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   "rules": {
     "indent": ["warn", 2, { "SwitchCase": 1, "MemberExpression": "off", "ignoredNodes": ["TemplateLiteral > *"] }],
     "linebreak-style": ["error","unix"],
+    "linebreak-style": ["error","windows"],
```

Comment:
That just breaks it on Linux and OSX and will break the CI build. Just change the unix one to "warn".

Reply:
Also, read this: https://eslint.org/docs/rules/linebreak-style. Git is changing them to crlf on checkout, so the lint fails. But, "warn","unix" is fine, even if you don't update your git config.

Reply:
Hi, @adrianfish;
Thank you for the information.
I've changed "error" for "warn".

Greetings